### PR TITLE
Upgrade node-forge to non vulnerable version

### LINKED ISSUE #48: node-forge@^0.10.0 is vulnerable
Any new version plan?

### PR DESCRIPTION
**Pull request by mkapra**
Fix #48

### CODE REVIEW

**Comment by mkapra** [comment]
I ran the tests and they were successful.

File: package-lock.json [hunk 1]
```diff
@@ -1,8 +1,1157 @@
 {
   "name": "selfsigned",
   "version": "1.10.12",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (apepper) [inline package-lock.json:4]: This project still seems to use `package-lock.json` version 1. Can you redo your changes with `npm` version 6 installed? This way the diff of `package-lock.json` should be much smaller.

**Review by apepper — COMMENTED** [review]
FEEDBACK (4ed3feb)

File: package-lock.json (hunk 1 above)
Comment (mkapra) [reply package-lock.json:4]: Oh sure :)

Comment (mkapra) [reply package-lock.json:4]: My local node 6 version does not update the `package-lock.json`. Any ideas?

Comment (apepper) [reply package-lock.json:4]: You need to start with a v1 file of `package-lock.json`. I'm not sure if npm 6 will downgrade manually.

Comment (mkapra) [reply package-lock.json:4]: > You need to start with a v1 file of `package-lock.json`. I'm not sure if npm 6 will downgrade manually.

I tried different things:

1. Removed the whole `package-lock.json` file -> File gets not created anymore
2. Manually replaced `lockfileVersion` by `1` -> Nothing happens

Before each try I removed the `node_modules` directory to start fresh.

**Comment by jfromaniello** [comment]
Thank you, merged and pushed as v1.10.13

**Comment by jfromaniello** [comment]
https://github.com/jfromaniello/selfsigned/issues/50#issuecomment-1010010700

I had to publish a new patch `v1.10.14` which is the same than `v1.10.12` because the updated version of node-forget breaks with node < 10.

This fix has been realeased as a major version `v2.0.0`, I also introduced engines node >= 10 rule.

**Pull request merged**